src/Deribit/Common/API/Instrument.jl

```diff
@@ -2,7 +2,9 @@ module Instrument
 
 export InstrumentQuery,
     InstrumentData,
-    instrument
+    instrument,
+    Currency,
+    InstrumentKind
```

Comment:
I propose to allocate it to a separate `export`
```suggestion
export Currency,
    InstrumentKind
```